components/cookie/banner/src/index.js

```diff
@@ -124,12 +131,17 @@ CookieBanner.defaultProps = {
   cookieKey: 'user-has-accepted-cookies',
   dismissOnScroll: true,
   dismissOnScrollThreshold: 0,
+  onChangeCookieBanner: isDisplayed => isDisplayed,
   iconClose: (
     <IconX svgClass="sui-CookieBanner-closeIcon" fill="#000" size={16} />
   )
 }
 
 CookieBanner.propTypes = {
+  /**
+   * onChangeCookieBanner is a function that return a boolean that means if cookieBanner is or isn't displayed.
+   */
+  onChangeCookieBanner: PropTypes.func,
```

Comment:
since this component is named `CookieBanner`, i would define `onChangeCookieBanner` as `onChange`. Otherwise that could be reduntant, don't you think?

Reply:
Yes, it is thanks mate :)

Reply:
i think you forgot to update the playground ; )